definitions.yaml

```diff
@@ -241,3 +241,26 @@ definitions:
       owner:
         type: string
         description: Name of the organization owning the cluster
+
+
+  # A request for an auth token
+  V4CreateAuthTokenRequest:
+    type: object
+    properties:
+      email:
+        type: string
+        description: Your email address
+      password_base64:
+        type: string
+        description: Your password as a base64 encoded string
+
+  # A response to a successful auth token request
+  V4CreateAuthTokenResponse:
+    type: object
+    properties:
+      auth-token:
+        type: string
+        description: The newly created API token
+      code:
```

Comment:
The successful response does not need a `code`.

Error responses use the `V4GenericResponse` type which has a code.